pkgs/os-specific/linux/nvidia-x11/builder.sh

```diff
@@ -147,6 +147,11 @@ installPhase() {
         fi
     fi
 
+    if [ -n "$firmware" ]; then
+        # Install the GSP firmware
```

Comment:
noob question: what is the GSP firmware?

Reply:
It's a riscv binary signed by nvidia that runs on a co-processor found on newer generations of NVIDIA GPUs, which can handle the GPU initialization and other secret sauces.